prime-router/settings/STLTs/IN/in-phd.yml

```diff
@@ -0,0 +1,100 @@
+- name: "in-phd"
+  description: "Indiana Public Health Department"
+  jurisdiction: "STATE"
+  stateCode: "IN"
+  countyName: null
+  filters: null
+  featureFlags: null
+  keys: null
+  senders: []
+  receivers:
+    - name: "full-elr"
+      organizationName: "in-phd"
+      topic: "full-elr"
+      customerStatus: "testing"
+      translation:
+        schemaName: "classpath:/metadata/hl7_mapping/ORU_R01/ORU_R01-base.yml"
```

Comment:
Do you have prime-router/src/main/resources/metadata/hl7_mapping/receivers/STLTs/IN/IN-receiver-transform.yml schema for this?